mozetl/clientsdaily/rollup.py

```diff
@@ -137,7 +146,13 @@ def main(date, input_bucket, input_prefix, output_bucket,
         month_frame = month_frame.where(clause)
     with_searches = extract_search_counts(month_frame)
     results = to_profile_day_aggregates(with_searches)
-    write_by_activity_day(results, date, output_bucket, output_prefix)
+    partition_count = get_partition_count_for_writing(bool(sample_id))
+    spark.conf.set(
+        "mapreduce.fileoutputcommitter.marksuccessfuljobs", "false"
+    )  # Don't write _SUCCESS files, which interfere w/ReDash discovery
```

Comment:
Is the issue with `_SUCCESS` files documented else where? Is this a hive/presto or athena issue? If so, a bug number would be nice.